### ISSUE TITLE
How to set learning rate for a single layer?

### ISSUE THREAD

**Issue by fengshikun** [dmlc/mxnet#1150]
I want to set a different learning rate for an special convolution layer，while other layers stay unchanged during training process. How can I get such `updater` to use separate learning rate for one layer? Thanks!

**Comment by piiswrong** [comment]
use Optimizer.set_lr_scale

**Comment by fengshikun** [comment]
@piiswrong Thanks!
